### ISSUE TITLE
Home Page - Empty column in Application/Registration table

### ISSUE THREAD

**Issue by keytie** [GrantThorntonETSI/eFile_Reimagined#568]
There is an empty column in the application/registration widget.  
![image](https://user-images.githubusercontent.com/45850524/64810318-2923b880-d569-11e9-82cf-36bbaa986391.png)

**Comment by dutchTender** [comment]
this is probably needs to be redesigned. this is the column for the delete
button for none submitted filings.

**Comment by keytie** [comment]
How about adding a column header to make it clear for now?
'Delete Draft'

**Comment by dutchTender** [comment]
Sounds like a good idea. I ll tell Jackie.

**Comment by JackieBabos** [comment]
I agree. @dutchTender are you going to make this change or do we need to have lynn?

**Comment by JackieBabos** [comment]
![image](https://user-images.githubusercontent.com/44168600/65185958-dabd6080-da25-11e9-9ccf-869d0a2f9d5d.png)
